include/rocksdb/advanced_compression.h

```diff
@@ -618,8 +642,14 @@ const std::shared_ptr<CompressionManager>& GetBuiltinV2CompressionManager();
 std::shared_ptr<CompressionManagerWrapper> CreateAutoSkipCompressionManager(
     std::shared_ptr<CompressionManager> wrapped = nullptr);
 // Creates CompressionManager designed for the CPU and IO cost aware compression
-// strategy
+// strategy that selects compression algorithm based on the io goal and the cpu
+// budget
 // EXPERIMENTAL
-std::shared_ptr<CompressionManagerWrapper> CreateCostAwareCompressionManager(
-    std::shared_ptr<CompressionManager> wrapped = nullptr);
+std::shared_ptr<CompressionManagerWrapper> CreateAutoTuneCompressionManager(
+    std::shared_ptr<CompressionManager> wrapped = nullptr,
+    std::shared_ptr<IOGoalCPUBudgetFactory> budget_factory = nullptr);
+
+std::shared_ptr<IOGoalCPUBudgetFactory> makeDefaultBudgetFactory(
+    double cpu_budget, double io_goal, double cpu_minbudget, double io_mingoal,
+    Options opt);
```

Comment:
Another incorrect function naming convention. Lack of document on things users need to know in order to use it.